src/compiler/crystal/tools/doc/html/css/style.css

```diff
@@ -264,7 +277,7 @@ body {
   padding: 0;
 }
 
-.superclass-hierarchy .superclass + .superclass::before {
+.superclass-hierarchy .superclass+.superclass::before {
```

Comment:
Personally, I find (all occurrences of) this change less readable.

Reply:
I can revert it - it was just the default formatting performed by VS Code.

Reply:
I'd leave it to the core team to decide. The rest of the changes are also quite subjective btw, as they depend heavily on your prettier configuration.

Reply:
I'm not sure I follow, other than the formatting this just adds table css and allowing splits between namespaces

Reply:
I meant the formatting-related ones.